### PR TITLE
Add the Oblige 7.70 changelog

### PR DESCRIPTION
**Pull request by rawr51919**
Add the Oblige 7.70 changelog

### CODE REVIEW

**Comment by caligari87** [comment]
Where are these changelogs from?

**Comment by rawr51919** [comment]
> 
> 
> Where are these changelogs from?

This one's from the main changelog here in this repo, at the top of CHANGES.txt.

**Comment by caligari87** [comment]
AH, I see what this is for now. Unfortunately this is exactly the opposite of [CONTRIBUTING.md](https://github.com/caligari87/Oblige/blob/master/CONTRIBUTING.md)

* Do not use the code repository as a corkboard (To-do's, ideas, wishlists, notes, Fix-Me's, etc).
  * Yes, Andrew did this in many places. No, that doesn't make it okay.
  * "Cleanup" patches to remove existing commentary will be accepted ONLY if the removed material is first documented on the Wiki.

Basically I want to *remove* all that stuff from the repo and document it on the wiki, not add to it. Besides, there's no reason to have changelogs duplicated in multiple places.
